bedrock/firefox/templates/firefox/new.html

```diff
@@ -93,12 +93,15 @@
     <div class="row">
       <img src="{{ static('img/firefox/new/firefox-logo.png') }}" alt="Firefox" />
       <h1 class="large">
-      {% if l10n_has_tag('fx10_independent') %}
+      {% if LANG == 'en-US' or l10n_has_tag('spring_release') %}
```

Comment:
@jpetto @flodolo sorry Jon, I gave you some bad info. Let's keep this JUST en-US only. I think it may eventually go to other locales, but the message needs to be vetted for a global audience before it can be shown.